src/Simplex/Chat/Protocol.hs

```diff
@@ -322,6 +326,10 @@ msgContentText = \case
   MCText t -> t
   MCLink {text} -> text
   MCImage {text} -> text
+  MCVoice {text, duration} ->
```

Comment:
Not sure about `msgContentText` as we usually only put text here (which is not ideal in case of images as well). Though with this one we do not even plan to allow to include any text at all